python/lsst/meas/deblender/baseline.py

```diff
@@ -1119,7 +1310,53 @@ def _overlap(xlo, xhi, xmin, xmax):
 
 
 def _handle_flux_at_edge(log, psffwhm, t1, tfoot, fp, maskedImage,
-                         x0, x1, y0, y1, psf, pk, sigma1, patchEdges):
+                         x0, x1, y0, y1, psf, pk, sigma1, patchEdges
+    ):
+    """
+    If a substantial amount of flux is found at the edge of the template,
```

Comment:
You have a way of asking complicated questions. :)

I think instead of answering this question directly I'm going to re-write the docstring for ``deblend`` to give a more thorough description of all of the deblender steps. I'll try to answer all of your questions there, since other people are likely to have the same questions.